erpnext/accounts/general_ledger.py

```diff
@@ -3,345 +3,511 @@
 
 from __future__ import unicode_literals
 import frappe, erpnext
-from frappe.utils import flt, cstr, cint, comma_and, today, getdate, formatdate, now
```

Comment:
This is **serious** damage by mistake.

You have replaced `erpnext/accounts/general_ledger.py` by `erpnext/accounts/report/general_ledger/general_ledger.py`